### DISCUSSION TITLE
Memory connectors and other experimental features

### DISCUSSION THREAD

**Discussion by deepinderdeol** [microsoft/semantic-kernel#5795] [Q&A] [answered]
Hi - I want to get a better understanding of approximately when the features currently marked experimental can be expected to be finalized/stable? Prototyping and experimenting with these is okay but to start planning for actual development we need a better understanding of timeline and which of these items will actually move out of experimental phase.

Thanks.

**Comment by sophialagerkranspandey** [comment, marked as answer]
Filters and kernel content types will be non-experimental by Build (May 21st). We are also prioritizing memory connectors and agents. Are there particular features you're looking for?

**Comment by deepinderdeol** [comment]
Thanks @sophialagerkranspandey.
We have done prototyping and use both memory-connectors/agents, so these are important. Will the currently supported memory-connectors continue to be supported?

**Comment by sophialagerkranspandey** [comment]
We are looking to refactor the current memory connectors and they'll likely change by Build but the underlying functionality will still be supported.

**Reply by deepinderdeol** [reply]
Thanks.
